src/app/trust-us/trust-us.component.ts

```diff
@@ -0,0 +1,13 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-trust-us',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './trust-us.component.html',
+  styleUrl: './trust-us.component.scss',
+})
+export class TrustUsComponent {
+  hoveredIndex: number = -1;
```

Comment:
wkorzystajmy tutaj tzw. sygnały, dzięki którym apka będzie znacznie bardziej wydajna i będziemy potencjalnie mogli robić bardziej elegancki i zrozumiały kod. 
https://angular.dev/guide/signals

Wiem że w tej apce nie ma to większego znaczenia, ale jeśli w przyszłości masz pracować na bardziej złożonych apkach to warto żebyś się tego nauczył.